js/myparcel.js

```diff
@@ -58,12 +58,15 @@ MyParcel = {
         MyParcel.bind();
     },
 
-    getPriceHtml: function (priceOfDeliveryOption) {
-
-        if (!priceOfDeliveryOption) {
-            var price = "";
+    getPriceHtml: function (priceOfDeliveryOptionOrig) {
+        if (!priceOfDeliveryOption || (typeof priceOfDeliveryOption !== 'string' && typeof priceOfDeliveryOption !== 'number')) {
```

Comment:
Ik denk dat het beter is om hier ook een console.error('...') te gebruiken. Dan kan de programmeur dit beter debuggen.